### PR TITLE
Added Wallet/Blockchain validations

### PR DESCRIPTION
**Pull request by dinesh-skyach**
Added Wallet/Blockchain validations

### CODE REVIEW

File: spec/models/blockchain_spec.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        height:                5000000,
+        min_confirmations:     4,
+        explorer_address:      '',
+        explorer_transaction:  '',
+        status:                'disabled'
+      }
+    end
+
+    it 'creates valid record' do
+      record = Blockchain.new(valid_attributes)
```
Comment (tamadamas) [inline spec/models/blockchain_spec.rb]: Why not FactoryBot?

File: spec/models/wallet_spec.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        currency_id:          'eth',
+        address:              '0x2b9fBC10EbAeEc28a8Fc10069C0BC29E45eBEB9C',
+        kind:                 'hot',
+        nsig:                  1,
+        status:               'active'
+      }
+    end
+
+    it 'creates valid record' do
+      record = Wallet.new(valid_attributes)
```
Comment (tamadamas) [inline spec/models/wallet_spec.rb]: Why not FactoryBot?

**Review by tamadamas — CHANGES_REQUESTED** [review]

**Comment by dinesh-skyach** [comment]
@rxx I have made the changes. Please check now.

**Review by tamadamas — APPROVED** [review]
Looks good

**Review by mod — APPROVED** [review]

**Pull request merged**